### DISCUSSION TITLE
Python issue personalization

### DISCUSSION THREAD

**Discussion by liam-strong** [etsy/open-api#1223] [Q&A]
Hello,

I've been immersed in a project involving mass publishing listings, and I'm currently tackling the aspect of accepting custom text along with personalized instructions. However, upon adding the data array for submission, I'm encountering an unexpected response from the API.

Data Array
```
"is_customizable": True,
"is_personalizable": True,
"personalization_is_required": True,
"personalization_instructions": "Custom text here",
```

Response from Etsy API:
```
"is_personalizable":true,"personalization_is_required":false,"personalization_char_count_max":null,"personalization_instructions":null
```

Anyone have any ideas?

**Comment by liam-strong** [comment]
Anyone got any ideas at all?

**Comment by itsDrew86** [comment]
I am having the same issue. Just spent all week working on mockups and writing code/updating my database tables to release a new product that's built around personalization. Only to find out that I can't set personalization_is_required to False or set the "personalization_instructions" via the API. 

Seeing that this issue was raised 4 months ago with no response is disheartening.
